docs/compatibility.md

```diff
@@ -436,5 +448,5 @@ When translating UDFs to Catalyst expressions, the supported UDF functions are l
 |                          | x.toArray                                                |
 |                          | lhs += rhs                                               |
 |                          | lhs :+ rhs                                               |
-| Method call              | Only if the method being called <ol><li>consists of operations supported by the UDF compiler, and</li><li>is one of the folllowing:<ul><li>a final method, or</li><li>a method in a final class, or</li><li>a method in a final object</li></ul></li></ol> |
```

Comment:
The intent of the original code was to produce a bulleted list which makes it much easier to read.  The new version no longer does that.  Was the old formatting causing problems in something other than Markdown?

Reply:
The formatting does not work in github pages markdown: https://nvidia.github.io/spark-rapids/docs/compatibility.html

Reply:
<img width="960" alt="Screen Shot 2021-01-22 at 9 08 53 AM" src="https://user-images.githubusercontent.com/7036315/105522009-8f5a3880-5c91-11eb-948d-98f5ea3fc11e.png">

Reply:
Sigh.  Github pages/Jekyll is why we can't have nice things.